src/module.api/poolpair.controller.ts

```diff
@@ -183,6 +186,13 @@ export class PoolPairController {
   ): Promise<BestSwapPathResult> {
     return await this.poolSwapPathService.getBestPath(fromTokenId, toTokenId)
   }
+
+  @Get('/dexprices')
+  async listPrices (
```

Comment:
```suggestion
  async listDexPrices (
```
better give them a same..